server/test/utils.ts

```diff
@@ -74,6 +72,8 @@ function getExpectedUserObject(user: User) {
 }
 
 export {
+  makeUserObject,
+  createManyUsers,
```

Comment:
I'd vote for `createUsers` over `createManyUsers`, personally.

Aside from that, are we happy with the distinction between `makeUserObject` and `createUser`? If not, any other suggestions? @timmyichen

Reply:
How about `generateUserObject`

This was chatGPTs suggestion: `generateRandomUserObjectWithOptionalProperties`

Reply:
@MainlyColors that was the other name [I was considering](https://github.com/timmyichen/dev-directory/pull/74#discussion_r1155095468) as well. 

ChatGPT clearly doesn't understand the idea of _human_ maintainable code. 😂

Reply:
sure i can change the name

Reply:
`createUsers` being similar to `createUser` is completely fine if it makes sense semantically - as in, if it's just doing multiple of `createUser`, then i think it makes sense.

the issue that i had with makeUser/createUser is that, to those not familiar with Laravel  conventions (TIL!), the difference between the two is ambiguous.

with createUser/createUsers, though similar in the name itself, the meaning is different in that one is the pluralized form of the other

Reply:
alrighty, no problemo